src/inverseRobot.js

```diff
@@ -24,7 +24,17 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const inverseObj = {};
+
+  for (const key in robot) {
+    if (inverseObj[robot[key]] in robot) {
+      return null;
+    }
```

Comment:
It's better to check if robot[key] value already exists in inversedRobot, not in robot
```suggestion
    if (inverseObj[robot[key]]) {
      return null;
    }
```